scripts/build/jq-insert.sh

```diff
@@ -56,5 +56,12 @@ if [ -z "$ARGV_PROPERTY" ] ||
 fi
 
 TEMPORARY_FILE="$ARGV_TEMPORARY_DIRECTORY/$(basename "$ARGV_FILE").TMP"
-cat "$ARGV_FILE" | jq ".$ARGV_PROPERTY=\"$ARGV_VALUE\"" > "$TEMPORARY_FILE"
+
+if [ "$ARGV_VALUE" == "true" ] || [ "$ARGV_VALUE" == "false" ]; then
```

Comment:
I guess this is basically testing value-is-not-a-string, i.e. if we ever wanted to set a numeric value, we'd need that to also follow this first `if` branch. I wonder if a better solution might be to require string values passed to `jq-insert.sh` to already be double-quoted? e.g. change https://github.com/resin-io/etcher/blob/master/Makefile#L220 to
```
-v "\"$(ANALYTICS_SENTRY_TOKEN)\"" \
```
?